src/app/footer/footer.component.ts

```diff
@@ -59,9 +48,17 @@ export class FooterComponent implements OnInit {
     return about.name;
   }
 
-  constructor() { }
+  constructor(private categoriesList: FooterService) { }
 
   ngOnInit(): void {
+    this.getFooterCategories();
   }
 
+  private getFooterCategories(){
+    this.categoriesList.getCategories().subscribe((data) =>
+      {
+        this.shopDepartments = data;
+        console.log(data);
```

Comment:
1. Remove console.log
2. Add loader
3. Add error condition

Reply:
@akashsinha13  Added loader and error condition.